### ISSUE TITLE
Dependency Error Updater should create issues in Core-eng for infra errors and in customer repos for repo related errors

### ISSUE THREAD

**Issue by epananth** [dotnet/arcade#7615]
- [ ] This issue is blocking 
- [ ] This issue is causing unreasonable pain

Currently dependency error updater opens issues in core-eng if the dependency update fails. 

We need a way to differentiate between infra error vs repo error, so that we can open issues in customer repos if they are repo specific errors. This will reduce triage noise. 

cc: @riarenas  @ChadNedzlek

**Comment by riarenas** [comment]
I think adding the ability to bucketize the errors into categories that we could assign to repos or ourselves would be useful, but I think it's a fairly substancial rework of how the error processor works right now.

**Comment by markwilkie** [comment]
This one seems to be getting out of control a bit.  @mmitche - thoughts on what to do with all of these dependency update errors?

I think the challenge here is to differentiate the noise from the real errors.

**Comment by adiaaida** [comment]
Update: from what we've talked about in triage, we actually want to move this to using grafana/alerts to let us know when there are issues. Is that correct @garath @riarenas? Should we rename this issue to better reflect what we want to do? Close this and create a new issue?

**Comment by riarenas** [comment]
Yes, I believe that's what we decided. 'm fine with either of those approaches, but still don't know where we would triage such work to. Dependency flow is complete means this won't be addressed for a while, and these issues seem to be causing annoyance for people. Just not sure if the annoyance warrants urgency?

**Comment by adiaaida** [comment]
Updated the title to reflect what we actually want to do, feel free to modify it if it doesn't make sense

**Comment by garath** [comment]
Would this fall in Operations work? @ilyas1974

**Comment by ilyas1974** [comment]
Do we have an open epic in this space? If so, it should be part of that epic. If we have completed work in this space and are looking to change something to make it more sustainable, it can be added to the Operations work.

**Comment by garath** [comment]
I think then the "Dependency Flow is Complete" is the right place for now.
